docset/windows/vamt/find-vamtmanagedmachine.md

```diff
@@ -69,6 +69,14 @@ PS C:\>Find-VamtManagedMachine -QueryType ldap -QueryValue "ldap://domainname/??
 This command searches for computers by using an LDAP query.
 The search results are restricted to the computers that have LabComp in their name.
 
+### Example 5: Search in a domain and store results in specified database
+```
+PS C:\>Find-VamtManagedMachine -QueryType ActiveDirectory -QueryValue "DomainName" -MachineFilter "labcomp*" -DbConnectionString "Data Source=localhost\SQLEXPRESS;Initial Catalog=VAMT;Integrated Security=True;MultipleActiveResultSets=True"
+```
+
+This command searches the specified Active Directory domain.
```

Comment:
```suggestion
This command searches for the specified Active Directory domain.
```

Reply:
Thanks for the suggestion. The command search in a domain so I believe it should either be:
This command searches the specified
or
This command searches in the specified
Since this is the same in all examples, I would prefer not to change it and let the author decide.